.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "0.12"
   - "4"
   - "6"
-  - "stable"
+  - "7"
```

Comment:
well, i think we should add them manually. wouldn't want to los node 7 testing when node 8 arrives would we?